ca/config.go

```diff
@@ -135,6 +135,13 @@ func (s *SecurityConfig) UpdateRootCA(cert, key []byte, certExpiry time.Duration
 	return err
 }
 
+// loadNewRootCA replaces the root CA with a new root CA
+func (s *SecurityConfig) loadNewRootCA(rootCA *RootCA) {
+	s.mu.Lock()
+	defer s.mu.Unlock()
+	s.rootCA = rootCA
+}
```

Comment:
I don't think non-leader renew their own certs. IIRC all of them are calling the manager for rotation, even though they are technically capable of doing it on their own.

I still think it's worth figuring out at least the worker root rotation story, but ideally we would make sure that this feature is solid for both workers and managers.